### PR TITLE
fix ci

### PR DESCRIPTION
**Pull request by zbeyens**
**Description**

See changesets.

### CODE REVIEW

File: apps/www/public/registry/index.json [hunk 1]
```diff
@@ -824,7 +824,7 @@
     ],
     "registryDependencies": [],
     "files": [
-      "plate-ui/toggle-element.tsx"
+      "plate-ui/link-element.tsx"
```
Comment (zbeyens) [inline apps/www/public/registry/index.json:827]: oops, not sure how this got changed

File: packages/toggle/src/types.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-import { buildToggleIndex } from './store';
+import { buildToggleIndex } from './toggle-controller-store';
 
 export interface TogglePlugin {
   // Options would go here
   // TODO a JOTAI layer in plate-core instead of relying on plugin options
-  openIds: Set<string>;
-  setOpenIds: (args_0: SetStateAction<Set<string>>) => void;
-  toggleIndex: ReturnType<typeof buildToggleIndex>;
+  openIds?: Set<string>;
```
Comment (zbeyens) [inline packages/toggle/src/types.ts:8]: TS constraint: all fields need to be optional

File: apps/www/public/registry/index.json (hunk 1 above)
Comment (Liboul) [reply apps/www/public/registry/index.json]: I might have done something wrong, I'm not too sure how the whole build registry thing works

**Comment by Liboul** [comment]
Thanks!

**Pull request merged**